discord/commands/context.py

```diff
@@ -215,10 +215,15 @@ def unselected_options(self) -> Optional[List[Option]]:
                 return self.command.options  # type: ignore
         return None
 
+    @property
+    def send_modal(self) -> None:
```

Comment:
Why is this typed to `None`

Reply:
Because I forgot this branch doesn't have that update to the underlying method. Will remove.